packages/admin/cms-admin/src/contentScope/Select.tsx

```diff
@@ -36,15 +48,40 @@ export default function ContentScopeSelect({
 
     const [searchValue, setSearchValue] = React.useState<string>("");
 
-    const filteredValues = searchable
-        ? values.filter(
-              (item) => item.value.toLowerCase().includes(searchValue.toLowerCase()) || item.label?.toLowerCase().includes(searchValue.toLowerCase()),
-          )
-        : values;
+    const filterBySearchValue = (
+        searchValue: string,
+    ): {
+        group: ContentScopeInterface;
+        groupKey: string;
+        values: ContentScopeInterface[];
+    }[] => {
+        if (!searchValue) return values;
+        const valuesWithId = values.map((value, index) => ({ id: index, ...value }));
+        const groupValues = valuesWithId.filter((scopeVal) => scopeVal.group?.value.includes(searchValue.toLowerCase()));
+
+        const valuesWithSearch = valuesWithId
+            .filter((scopeVal) => !scopeVal.group?.value.includes(searchValue.toLowerCase()))
+            .map((scopeVal) => {
+                return {
+                    ...scopeVal,
+                    values: scopeVal.values.filter((val) => {
+                        return Object.keys(val)
+                            .map((key) => val[key])
+                            .some(({ label, value }: { value: string; label?: string }) => {
+                                return value.includes(searchValue.toLowerCase()) || label?.toLowerCase().includes(searchValue.toLowerCase());
+                            });
+                    }),
+                };
+            })
+            .filter((scopeVal) => scopeVal.values.length > 0);
 
+        return [...groupValues, ...valuesWithSearch].sort((a, b) => a.id - b.id);
```

Comment:
Why do you need this? Why does the order ever change?

Reply:
When searching i filter all the values with the searchterm in the 1. Dimension. This can sometimes cause the array to change its order.